eng/pipelines/templates/stages/archetype-sdk-tool-dotnet.yml

```diff
@@ -1,9 +1,15 @@
 resources:
   repositories:
+    - repository: azure-sdk-for-net
```

Comment:
Do we really want to reference the azure-sdk-for-net repo? Is that the correct direction?

Reply:
It feels like we should either have the template in this repo or in the build-tools repo.